submissions/pashapushak1996/html-css-popup/scss/_layout.scss

```diff
@@ -0,0 +1,239 @@
+@import "base";
+
+body {
+  background-color: $white;
+  color: $grey-2;
+}
+
+.wrapper {
+  min-height: 100%;
+}
+
+.container {
+  max-width: 1200px;
+
```

Comment:
please remove unnecessary empty lines across the file

Reply:
I did this.